content/pages/kits.md

```diff
@@ -0,0 +1,72 @@
+Title: Ordering, Building, and Operating DIYnamics Kits
+Slug: kits
+
+## The DIYnamics Kits
+
+The DIYnamics Project provides the resources for you to order, build and operate your own rotating tank platform! As detailed in the [About](/pages/about.html) page, these devices are used to model geophysical fluid dynamics. Rotating tanks of water feature fluid behavior analogous to planetary fluid bodies fundamentally impacted by rotation. As part of our Do-It-Yourself approach, our goal is for educators to perform experiments with rotating tank platforms themselves. Hands-on demonstrations provide a valuable opportunity for students to ask questions and draw conclusions in linking experimental results to real world phenomena.
+
+The DIYnamics Kits provide robust rotating tank platforms at various price points. Previously, obtaining your own device required an often custom design and complicated construction process. We developed our family of models to provide cheap, simple and robust solutions for your own device. Each of the below models is well suited for performing fluid dynamics experiments with students. The rotating tank platforms in the hierarchy of models differ in their ability to scale up in terms of speed and rotation rate.
+
+---
+
+## LEGO-Based Tables - Starting at $60
+
+{% img {static}../images/kits/lego/technics_table.PNG 400 Technics Table complete setup %}
+
+The LEGO-Based Tables are 16-inch diameter rotating tank platforms built using LEGOs, a lazy susan and other household items. The tables are perfect for providing an introduction to geophysical fluid dynamics and are safe for audiences of all ages. The LEGO-Based Tables are cheap yet robust enough for a wide range of experiments and available add-ons enable capturing rotating frame footage and changing rotation rate.
+
+### [Learn more about the LEGO-Based Tables here](/pages/lego.html)
+
+---
+
+## DJ Table - Starting at $1000
+
+{% img {static}../images/kits/dj/dj_table.PNG 400 DJ Table complete setup %}
+
+​The DJ Table is a record player based system for demonstrating geophysical fluid dynamics. The system’s tanks come with transparent lids that make them ideal for visualization via phone or camera, and its minimal footprint makes for easy storage and transportation.​ ​
```

Comment:
Add:

Compared to the LEGO-based tables, the DJ Table provides much finer control over the rotation rate and a more reliable, consistent rotation rate.  This makes it great for demonstrations such as Taylor Columns that require precise changes in rotation rate.  While not quite as portable as the LEGO-based tables, the DJ Table's rolling cart still makes it highly portable.